2021/_data/talks.json

```diff
@@ -262,7 +320,7 @@
+      "abstract": "'Entropy is the price of a Structure '. But sometimes, the price turns too high to pay. Is your codebase turning untestable, unreadable, and unmaintainable? Does your costs to add new features aren't meeting the ROIs. It's time to ask your manager to fund **Refactoring**!\n    - In this demo-driven talk, using a real-world service, we shall identity some common reasons for a codebase to rot and how to weed them out using principles from **Clean Architecture** and **Functional Programming**, along with achieving 95% test coverage and ~50% reduction in [Cognitive complexity](https://www.sonarsource.com/docs/CognitiveComplexity.pdf).\n\n Audience & Takeaways\nThis applies to software developers at all levels. I use **Java** to demonstrate the code snippets, but this talk is about Software Design and Testing and is agnostic of programming language. The major takeaways for the audience are:\n    - Pragmatically apply the principles of Clean-Architecture and Functional programming in real-world refactoring.\n    - How to break your codebase into atomic components which can be composed and tested-in-isolation.\n    - How to eliminate _ Exceptions_ from your code using **ADTs (Algebraic Data Types)**.\n    - How to replace _Shared Global mutable state on your Shared Code_ with **Immutability** and **Data Transformations**.\n    - How separating out _What-to-do_ from _How-to-do_ can bring in a lot of flexibility to your code-base.\n    - How following all the above organically boost your **Testability** without falling into any anti-patterns of testing.\n    - Let's talk about **Complexity** and how to objectively measure the reduction we brought with our refactoring.\n",
```

Comment:
I don't know if we should remove what we have after: `Audience & Takeaways`.... what is included afterwards it should be hidden I guess...
![image](https://user-images.githubusercontent.com/2370634/123830055-3f983100-d903-11eb-8c17-7d7de8dd4d8b.png)

Reply:
Yes, I agree it should be removed. Could you create a code suggestion?